partner_contact_nationality/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (C) 2014-2015  Grupo ESOC <www.grupoesoc.es>
+# © 2017-Apertoso N.V. (<http://www.apertoso.be>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    "name": "Contact nationality",
+    "summary": "Add nationality field to contacts",
+    'version': '11.0.1.0.0',
+    "category": "Customer Relationship Management",
+    "website": "https://odoo-community.org/",
```

Comment:
@jprobineau IMHO Please change URL as described in https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L10